src/plugins/lanplus/lanplus.c

```diff
@@ -1868,7 +1869,7 @@ ipmi_lanplus_build_v2x_ipmi_cmd(struct ipmi_intf *intf, struct ipmi_rq *req,
 	 */
 	static uint8_t curr_seq = 0;
 
-	if (isRetry == 0)
+	if (isRetry == false)
 		curr_seq += 1;
 
 	if (curr_seq >= 64)
```

Comment:
What is `64`, by the way? Do you know? Looks like some maximum possible sequence number (probably from IPMI spec).